.github/workflows/scripts/e2e.gradle.default.verify.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# shellcheck source=/dev/null
+source "./.github/workflows/scripts/e2e-verify.common.sh"
+
+# Input variables
+EXPECTED_ARTIFACT_OUTPUT=${EXPECTED_ARTIFACT_OUTPUT:-}
+PROVENANCE_DIR=${PROVENANCE_DIR:-}
+GITHUB_REF_NAME=${GITHUB_REF_NAME:-}
+GITHUB_REF=${GITHUB_REF:-}
+GITHUB_REF_TYPE=${GITHUB_REF_TYPE:-}
+RUNNER_DEBUG=${RUNNER_DEBUG:-}
+PROJECT_DIR=${PROJECT_DIR:-}
+if [[ -n "${RUNNER_DEBUG}" ]]; then
+    set -x
+fi
+
+cp -r  "slsa-attestations" "${PROJECT_DIR}/"
+cp -r build "${PROJECT_DIR}/"
+cd "${PROJECT_DIR}" || exit # exit to satisfy Shellcheck
+
+artifact_version=$(./gradlew properties -q | grep "version:" | awk '{print $2}')
+artifact_id=$(./gradlew properties -q | grep "name:" | awk '{print $2}')
+artifact_name="${artifact_id}-${artifact_version}.jar"
+provenance="${PROVENANCE_DIR}/${artifact_name}.build.slsa"
+
+go env -w GOFLAGS=-mod=mod
+
```

Comment:
See changed https://github.com/slsa-framework/example-package/blob/main/.github/workflows/scripts/e2e.maven.default.verify.sh#L18-L49

Reply:
not just colors. Also I needed to set the BINARY and PROVENANCE env variables. `go env -w GOFLAGS=-mod=mod` is not needed

Reply:
binary and provenance need to be capital letters

Reply:
Thanks.